forge-ai/src/main/java/forge/ai/SpecialCardAi.java

```diff
@@ -1610,6 +1610,22 @@ public boolean apply(Card card) {
         }
     }
 
+    // The One Ring
+    public static class TheOneRing {
+        public static boolean consider(final Player ai, final SpellAbility sa) {
+            if (!ai.canLoseLife() || ai.cantLoseForZeroOrLessLife()) {
+                return true;
+            }
+
+            AiController aic = ((PlayerControllerAi) ai.getController()).getAi();
+            int lifeInDanger = aic.getIntProperty(AiProps.AI_IN_DANGER_THRESHOLD);
+            int numCtrs = sa.getHostCard().getCounters(CounterEnumType.BURDEN);
+            boolean notDangerous = !ai.canLoseLife() || ai.cantLoseForZeroOrLessLife() || (ai.getLife() > numCtrs + 1 && ai.getLife() > lifeInDanger);
```

Comment:
aren't the first two conditions already checked above anyway?

Reply:
Good catch! Done :)